pages/nft/url/index.vue

```diff
@@ -240,13 +239,15 @@ export default class FetchIndex extends Vue {
         return
       }
       this.isLoading = true
+      this.arweaveId = ''
+      this.arweaveFeeTxHash = ''
       await this.crawlUrlData()
       if (this.crawledData?.body) {
         try {
           const arweaveFeeInfo = await this.estimateArweaveFee()
           if (!this.arweaveId) {
-            const txHash = await this.sendArweaveFeeTx(arweaveFeeInfo)
-            await this.submitToArweave(txHash)
+            await this.sendArweaveFeeTx(arweaveFeeInfo)
+            await this.submitToArweave()
```

Comment:
Should we skip `sendArweaveFeeTx` if `arweaveFeeTxHash ` is set?
Would `arweaveFeeTxHash ` reset if we get api error that thinks its invalid?

Similarly, would we skip `crawlUrlData` if `this.crawledData?.body` exists?